foundry.lock

```diff
@@ -0,0 +1,11 @@
+{
+  "lib/eigenlayer-contracts": {
+    "rev": "64c028333856ab2ae603fcde94fc9b24715ef603"
```

Comment:
Just checking: Should we bump up the core contracts repo commit? Seems like this commit is from early Feb. I don't think it matters much since it's only interacting with bEigen and the RewardsCoordinator which shouldn't have changed since then.